.vale/styles/RedHat/SelfReferentialText.yml

```diff
@@ -12,3 +12,11 @@ tokens:
   - this chapter
   - this section
   - this subsection
+  - The following
```

Comment:
https://www.ibm.com/docs/en/ibm-style?topic=word-usage#following

Can we make this more useful by creating a sequence rule that returns instances of "following" where it is not in the adjectival position before a noun? 

https://github.com/aireilly/vale-at-red-hat/blob/add-pos-using-rule/.vale/styles/RedHat/Using.yml

https://vale.sh/docs/topics/styles/#sequence

Hint: